frontend/src/components/organisms/ProfileForm.tsx

```diff
@@ -40,8 +50,102 @@ interface ProfileFormProps {
   userDetails: formData;
 }
 
+interface ModalBodyProps {
+  userDetails: formData;
+  handleClose: () => void;
+}
+
+/** Confirmation Modal To Delete An Account */
+const ModalBody = ({ userDetails, handleClose }: ModalBodyProps) => {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<DeleteAccountFormValues>();
+
+  //changeUserStatus handles deleting user from prisma and firebase
+  const { mutateAsync: changeUserStatus, error: changeUserStatusError } =
```

Comment:
You should use appropriate naming patterns. I don't think `changeUserStatus` is the way